myapp/models.py

```diff
@@ -3,16 +3,16 @@
 from django.urls import reverse
 
 
-
-#base model
+# base model
 class BaseModel(models.Model):
     created_at = models.DateTimeField(default=django.utils.timezone.now())
     updated_at = models.DateTimeField(default=django.utils.timezone.now())
```

Comment:
Maybe you can use `auto_now=True` for `created_at` and `auto_now_add=True` for `updated_at`. Please check before modifying.